scss/titles.scss

```diff
@@ -15,7 +15,26 @@
   .title {
     color: black;
     font-size: $text-big;
-    left: 420px;
-    position: absolute;
+    position: relative;
+
+    &::before {
+      background: url('maquetado/assets/triangle.svg');
```

Comment:
Move the attributes `background-repeat` and `background-size` to `background`. Remember it is a *shorthand*: [CSS background Property](https://www.w3schools.com/cssref/css3_pr_background.asp)